hotline/notifications/views.py

```diff
@@ -25,3 +25,18 @@ def create(request):
     return render(request, "notifications/create.html", {
         "form": form,
     })
+
+
+@permissions.is_active
+def list_(request):
+    if request.method == "POST":
+        form = UserSubscriptionDeleteForm(request.POST, user=request.user)
+        if form.is_valid():
+            form.save()
```

Comment:
The latter one is correct. You want to leave a message after `form.is_valid()` is checked for truyness. I wouldn't add a message when the form is invalid, since you already inform them on the template.